### PR TITLE
Fix env vars and add UI url

### PR DESCRIPTION
**Pull request by qwerty287**
Closes https://github.com/woodpecker-ci/woodpecker/issues/2219

### CODE REVIEW

File: pipeline/frontend/metadata/environment.go [hunk 1]
```diff
@@ -97,7 +100,8 @@ func (m *Metadata) Environ() map[string]string {
 		"CI_PREV_PIPELINE_NUMBER":        strconv.FormatInt(m.Prev.Number, 10),
 		"CI_PREV_PIPELINE_PARENT":        strconv.FormatInt(m.Prev.Parent, 10),
 		"CI_PREV_PIPELINE_EVENT":         m.Prev.Event,
-		"CI_PREV_PIPELINE_URL":           m.Prev.Link,
+		"CI_PREV_PIPELINE_URL":           m.getPipelineStatusLink(m.Prev, 0),
+		"CI_PREV_PIPELINE_FORGE_URL":     m.Curr.Link,
```
Comment (anbraten) [inline pipeline/frontend/metadata/environment.go:104]:
```suggestion
		"CI_PREV_PIPELINE_FORGE_URL":     m.Prev.Link,
```

File: pipeline/frontend/metadata/environment.go [hunk 2]
```diff
@@ -123,3 +127,11 @@ func (m *Metadata) Environ() map[string]string {
 
 	return params
 }
+
+func (m *Metadata) getPipelineStatusLink(pipeline Pipeline, stepNumber int) string {
```
Comment (anbraten) [inline pipeline/frontend/metadata/environment.go:131]: Status sounds a bit misleading to me. Maybe sth like `getPipelineUILink`

File: docs/docs/20-usage/50-environment.md [hunk 3]
```diff
@@ -80,7 +80,8 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_PIPELINE_NUMBER`             | pipeline number                                                                              |
 | `CI_PIPELINE_PARENT`             | number of parent pipeline                                                                    |
 | `CI_PIPELINE_EVENT`              | pipeline event (push, pull_request, tag, deployment)                                         |
-| `CI_PIPELINE_URL`                | link to the forge's web UI for the commit(s) or tag that triggered the pipeline              |
+| `CI_PIPELINE_URL`                | link to the web UI for the pipeline                                                          |
+| `CI_PIPELINE_FORGE_URL`          | link to the forge's web UI for the commit(s) or tag that triggered the pipeline              |
```
Comment (lafriks) [inline docs/docs/20-usage/50-environment.md:84]: Why do we need it if we already have `CI_COMMIT_URL`?

Comment (xoxys) [reply docs/docs/20-usage/50-environment.md:84]: As done with other env vars in the past, deprecated it and remove it in v3?

**Comment by xoxys** [comment]
Do we want to add a deprecation for `CI_PIPELINE_URL` as well or do we skip it as it was broken anyway?

File: docs/docs/20-usage/50-environment.md [hunk 4]
```diff
@@ -110,6 +111,7 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_PREV_PIPELINE_PARENT`        | previous pipeline number of parent pipeline                                                  |
 | `CI_PREV_PIPELINE_EVENT`         | previous pipeline event (push, pull_request, tag, deployment)                                |
 | `CI_PREV_PIPELINE_URL`           | previous pipeline link in CI                                                                 |
```
Comment (xoxys) [inline docs/docs/20-usage/50-environment.md:113]: I'm a bit confused. `CI_PIPELINE_URL` was removed, but we keep `CI_PREV_PIPELINE_URL` while the new one `CI_PREV_PIPELINE_FORGE_URL` was added here as well. Is this intended?

**Review by xoxys — APPROVED** [review]

**Pull request merged**